frontend/apps/erd-sample/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@liam-hq/erd-sample",
+  "private": true,
+  "version": "0.0.0",
+  "devDependencies": {
+    "@liam-hq/cli": "workspace:*"
+  },
+  "scripts": {
+    "build": "node ../../packages/cli/dist-cli/bin/cli.js erd build --input ../../packages/cli/fixtures/input.schema.rb --format schemarb"
+  }
+}
```

Comment:
I see...
Given dogfooding, the need to create packages for build and deploy is a bad user experience and should be avoided.
But this could be improved separately. ref: https://github.com/liam-hq/liam/pull/171/files#r1872472533

Reply:
Ok, since this doesn’t take much time for now, I was a bit unsure about the decision :)